core/src/main/java/com/klaytn/caver/contract/Contract.java

```diff
@@ -159,17 +190,17 @@ public Contract deploy(ContractDeployParams deployParam, SendOptions sendOptions
         String input = ABI.encodeContractDeploy(this.getConstructor(), deployParam.getBytecode(), deployParam.getDeployParams());
 
         SmartContractDeploy smartContractDeploy = new SmartContractDeploy.Builder()
-                .setKlaytnCall(caver.rpc.klay)
+                .setKlaytnCall(rpc.klay)
```

Comment:
```suggestion
                .setKlaytnCall(getRpc().klay)
```
I think it would be better to use the same style.